tiingo/api.py

```diff
@@ -144,7 +182,7 @@ def get_ticker_price(self, ticker,
                 fmt (string): 'csv' or 'json'
                 frequency (string): Resample frequency
         """
-        url = "tiingo/daily/{}/prices".format(ticker)
+        url = self._get_url(frequency).format(ticker)
```

Comment:
How would you feel about letting `_get_url` consume `ticker` as an argument? It might be more intuitive based on reading the function name that a string is returned rather than a string template that needs to have "format" called on it. If we leave this as is, we should at least update the docstring for `_get_url` to indicate it returns a string template rather than a string that can be used right away.

Reply:
Much better - changed.